using_middleware/src/hello_world.app.src

```diff
@@ -0,0 +1,16 @@
+{application, hello_world,
+ [{description, "An OTP application"},
+  {vsn, "0.1.0"},
+  {registered, []},
+  {mod, { hello_world_app, []}},
+  {applications,
+   [kernel,
+    stdlib
```

Comment:
Have you tried adding `elli` to this list to fix your issue?

Reply:
Doesn't seem to do the trick